examples/kafka/kafka-ephemeral.yaml

```diff
@@ -7,7 +7,15 @@ spec:
     version: 2.3.0
     replicas: 3
     listeners:
-      plain: {}
+      external:
+        type: loadbalancer
+        configuration:
+          serverKey:
+            secretName: my-cluster-kafka-brokers
+            certificate: my-cluster-kafka-0.crt
+            key: my-cluster-kafka-0.key
+        authentication:
+          type: tls
```

Comment:
I think this example should stay simple without `external` and any other "complication" with certs secrets related to it.

Reply:
Right... commited by an accident.